--- libiscsi-1.19.0/lib/Makefile.am.orig	2019-07-14 00:04:44.000000000 +0200
+++ libiscsi-1.19.0/lib/Makefile.am	2019-10-19 20:13:59.679343733 +0200
@@ -5,6 +5,7 @@
 	login.c nop.c pdu.c iscsi-command.c \
 	scsi-lowlevel.c socket.c sync.c task_mgmt.c \
 	logging.c
+libiscsi_la_LIBADD =
 
 if !HAVE_LIBGCRYPT
 libiscsi_la_SOURCES += md5.c
@@ -12,6 +13,7 @@
 
 if HAVE_LINUX_ISER
 libiscsi_la_SOURCES += iser.c
+libiscsi_la_LIBADD += -lpthread
 endif
 
 SOCURRENT=9
@@ -22,7 +24,7 @@
 	-no-undefined -export-symbols ${srcdir}/libiscsi.syms
 
 if HAVE_LINUX_ISER
-libiscsi_la_LDFLAGS += -libverbs -lrdmacm
+libiscsi_la_LIBADD += -libverbs -lrdmacm
 endif
 
 libiscsi_la_CPPFLAGS = -I${srcdir}/../include -I$(srcdir)/include \
